### ISSUE TITLE
Investigate replacing custom commands with drupal/console commands

### ISSUE THREAD

**Issue by grasmash** [acquia/blt#251]
setup:drupal:install <=> site:new
setup:drupal:settings
setup:update <=> update:execute
tests:phpunit <=> test:run
tests:behat <=> we have a PR for that

Pinging @jmoliveras

**Comment by grasmash** [comment]
@jmolivas I'd like to dig in a bit deeper to determine if each of these has parity with our existing targets. If the answer in "no" for any of these, let me know if you'd like to add the capability to the console command, or if it is out of scope.

* site:new. Does this do all of the following?
    * chmod sites/default to be writable
    * install drupal w/ sitename, site mail, username, pw, profile, and install_configure_form options
    * enable config module
    * import configuration
* setup:update. Does this do all of the following?
    * execute database updates  
    * import config
    * revert features
    * rebuild caches
* test:run. Does this do all of the following?
    * Execute all phpunit tests in a predefined directory
    * Output results to parsable xml file

For any operations not currently performed by the console commands, we'd need to either submit a PR to console to introduce the functionality, or else continue to use Phing targets.

**Comment by jmolivas** [comment]
@grasmash  I am completing some changes on DC  for RC-1, decoupling the project into separated projects.

* Drupal Console Languages, first repo created was English, more to come.
   https://github.com/hechoendrupal/drupal-console-en

* Generic commands and features to be shared across projects.
   https://github.com/hechoendrupal/drupal-console-core

* Main repo, this is the one you will be installing on your site via composer.
   https://github.com/hechoendrupal/DrupalConsole/pull/2583

* New global executable. This will be the phar you download when using curl.
   https://github.com/hechoendrupal/drupal-console-launcher

More info at:
http://jmolivas.com/drupalconsole-rc-1-release-close-a-lot-of-changes

Some commands will be available at Launcher and some other at the main repo.

Once this update is done and RC-1 is release ETA end of August 2016. 

I will start working on introduce the missing functionality to DC:
* chmod sites/default to be writable
* install drupal w/  install_configure_form options

* revert features

* Execute all phpunit tests in a predefined directory
* Output results to parsable xml file

**Comment by jmolivas** [comment]
@grasmash can you jump into our gitter channel to talk in detail about scheduling a call?.

We have most of that functionality in place using DC commands and custom chain commands

**Comment by jmolivas** [comment]
@grasmash this is an example we run it like `bin/drupal site:install:local:develop` and the config and commands are:
```
command:
  name: site:install:local:develop
  description: 'Install local development site'
commands:
# Install site
  - command: site:install
    options:
      force: true
      uri: 'http://default'
    arguments:
      profile: lightning
# Install development modules
  - command: module:install
    arguments:
      module:
        - features_ui
# Create dummy data
  - command: 'create:terms'
    options:
        limit: '10'
        name-words: '3'
    arguments:
        vocabularies:
            - tags
  - command: 'create:nodes'
    options:
        limit: 25
        title-words: 5
        time-range: 2592000
    arguments:
        content-types:
            - articles
# Add write permissions
  - command: exec
    arguments:
      bin: chmod +w sites/default/
  - command: exec
    arguments:
      bin: chmod +w sites/default/settings.php
```
